internal/platform/mmap_unsupported.go

```diff
@@ -13,10 +14,10 @@ func munmapCodeSegment(code []byte) error {
 	panic(errUnsupported)
 }
 
-func mmapCodeSegmentAMD64(code []byte) ([]byte, error) {
+func mmapCodeSegmentAMD64(code io.Reader, size int) ([]byte, error) {
```

Comment:
this has been drifted and hasn't been tested, so added the check incide `make check` recipe